tests/foreman/cli/test_user.py

```diff
@@ -545,3 +545,46 @@ def test_custom_personal_access_token_role(self, target_sat):
             f'curl -k -u {user["login"]}:{token_value} {target_sat.url}/api/v2/users'
         )
         assert f'Unable to authenticate user {user["login"]}' in command_output.stdout
+
+    @pytest.mark.tier2
+    def test_negative_personal_access_token_invalid_date(self, target_sat):
+        """Personal access token with invalid expire date.
+
+        :id: 8c7c91c5-f6d9-4709-857c-6a875db41b88
+
+        :steps:
+            1. Set the expired time to a nonsensical datetime
+            2. Set the expired time to a past datetime
+
+        :expectedresults: token is not created with invalid or past expire time
+
+        :CaseLevel: System
+
+        :CaseImportance: Medium
+
+        :BZ: 2231814
+        """
+        user = make_user()
+        User.add_role({'login': user['login'], 'role': 'Viewer'})
```

Comment:
As you already calling target_sat fixture, can you update this to use robottelo cli and cli_factory?